src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java

```diff
@@ -82,14 +107,30 @@ private boolean tasksFinished(JDFJob job) {
         return job.getTasks().size() == tasksFinished;
     }
 
+    private CommandState getCommandState(ArrebolCommandState arrebolCommandState) {
+        switch (arrebolCommandState) {
+            case RUNNING:
+            case UNSTARTED:
```

Comment:
Why UNSTARTED should be mapped to RUNNING state?

Reply:
Maybe PENDING or literally UNSTARTED will be better.